EU4toV2/Data_Files/blankMod/output/decisions/converterUnions.txt

```diff
@@ -3423,6 +3446,309 @@ political_decisions = {
 				civilized = yes
 				capital = 2068 
 			}
+			any_country = { limit = { tag = AAA } NOT = { exists = AAA } } primary_culture = dummy religion = pre_schism government = theocracy civilized = no capital = 322 } # AAA
```

Comment:
What is all of this about?

Reply:
Besides "AAA", these following ones are tags that are forced to be in the Vic2 mod but may not get their history from EU4. Some of them get cores on start due to cultural conversion or decisions and it'd break immersion to see, e.g., Sami peoples creating a British-led Sami country.

Reply:
Okay. Sounds like a bigger thing we should clean up directly, but it sounds like this is a good solution.

Reply:
Well, I kept doing it for a lot of tags (e.g. HRE or Longobardia), but this solved all the ones I was missing